src/qz/printer/status/StatusMonitor.java

```diff
@@ -68,6 +69,8 @@ public synchronized static void closeNotificationThreads() {
     }
 
     public synchronized static boolean startListening(SocketConnection connection, JSONArray printerNames) {
+        if (SystemUtilities.isMac()) CupsUtils.convertPrinterNames(printerNames);
```

Comment:
Is this method/call necessary? Can't we just pull the lookup here and use the values from it directly in this method's loop instead?